### ISSUE TITLE
[a11y Design Review] Benefits and Claims Non-Disability Experience: Burial Form Updates: Review Sketch Files

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#69302]
## Auditor(s)

@SarahKay8 
@sara-amanda 

## About the Benefits and Claims Team

<details><summary>Toggle to view team info.</summary>

- Team name: Benefits and Claims | Lifestage Benefits Crew | Non-Disability Experience
- OCTO-DE product owner: Emily Theis
- Product manager: Laura Steele
- Designer: Fiorella Glavez
- Accessibility specialist: none
- Content writer: none
- **Product/team Slack channel**: [#benefits-non-disability ](https://dsva.slack.com/archives/C057W3N9K2S)
- **Related Epic from the Collaboration Cycle:** #65251
</details>

## Important Notes
- **Trying to get MVP out fast** _with future enhancements_.
- **Just want us to look at added fields here.**
- **Pension painpoint is more accute**, Laura Steele confirmed with Emily this should be priority
    - _Burial forms are at least being processed, but looking to update._
    - **Intake Ticket Note:** Fixing the 530 form is a high-priority for OCTO and the VBA.

## Background Information
- Overview information collected during `CAIA Kickoff`. 
    - The [kickoff meeting was held Nov. 6, 2023](https://github.com/department-of-veterans-affairs/va.gov-team/edit/master/teams/CAIA/meeting-notes/2022-11-06-Kickoff-Meeting-Burials-Form-Updates-Non-Disability-Experience-Team.md). 
- Additional background information extracted from:
    - The `Governance Platform Design Intent a11y Ticket`
    - The `Initial Request Description` in CAIA's intake ticket.

### Review background information 

<details><summary>Toggle for a brief overview of initial a11y considerations</summary>

### Some Initial A11y Considerations:
**Sketch screens are ready for review**, except for the questions they flagged for SMEs, Julie can let us know about any of those.

</details>

<details><summary>Toggle to review Governance Design Intent Accessibility Feedback</summary>

### Should

> Wherever possible, try to use a "one thing per page" approach (which you mentioned you are doing, so that is good to see) rather than any sort of conditional display of information or form fields. There are a handful of situations where conditionally revealing new things makes sense, but usually breaking things into separate pages helps with cognitive load (keeps focus on a particular task) and avoids some tricky assistive technology interactions.
> As mentioned in the meeting, some small ones that are tightly coupled together are usually fine. And as Matt said, when it does come time for coding, you will want to make sure any new information is correctly announced for assistive technology users.

</details>

<details><summary>Toggle to review the original request description in CAIA's intake</summary>

## CAIA Ticket Request - Original Description

> We would like CAIA to review the critical updates we are making to the 530EZ burial form, so it can be successfully processed.

> We ask that the scope of CAIA's review be limited to the form pages we are touching, so we can quickly release a working MVP version of the updated form. Future work will focus on taking a more comprehensive look at the form to determine additional enhancements, replace depreciated design components, and address accessibility issues.

> We intend to fix defect-0 and defect-1 level A11y errors, but we would like to limit those fixes to the pages we are touching with these field updates.
The current online form is missing critical fields that were added to the paper form over the last 2 years. These missing fields are causing significant processing delays and putting additional burden on Veterans.

> The changes include adding missing fields and removing depreciated fields.

>Fixing the 530 form is a high-priority for OCTO and the VBA.

</details>

## Prototype 🔍 
- **Includes links to each step, along with:**
    - What is new
    - What has changed
    - What is not changing
    - Problems identified
  
### Prototype Links & Info. 👀 
<details><summary>Toggle to View Prototype Links</summary>

### Test with Sketch Prototypes
[Prototype or mockups](https://sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7)

- Step [1](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/350AC352-727B-4D99-B72B-D466886BE199/canvas) 
- Step [2](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/4871EF16-DFC9-460E-AA1B-7ED24E33B188/canvas)
- Step [3](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/99FEF874-74D9-41F1-BD5E-AE7E7E1D768C/canvas)
- Step [4](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/B7F2215E-001C-41F3-B803-9DFD96E7B129/canvas)
- Step [5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/E1B6C12E-B2F8-4A50-9CC1-1F44853FA3C6/canvas) 
- Step [6](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/28963E25-18C3-4CDA-A356-BF4FD6A7A5A6/canvas)

### What is New, Changing (and Not)
`Notes from KO 11/6/2023`
- **New fields added** in Steps [1](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/350AC352-727B-4D99-B72B-D466886BE199/canvas) and [2](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/4871EF16-DFC9-460E-AA1B-7ED24E33B188/canvas), a few in Step [3](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/99FEF874-74D9-41F1-BD5E-AE7E7E1D768C/canvas).
- **New page**:Step [4](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/B7F2215E-001C-41F3-B803-9DFD96E7B129/canvas) for transportation allowance.
- **No major changes** to Step [5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/E1B6C12E-B2F8-4A50-9CC1-1F44853FA3C6/canvas) or [6](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/28963E25-18C3-4CDA-A356-BF4FD6A7A5A6/canvas).
- **Changes pending**
    - **Fiorella** has set up current and proposed flows in sketch, annotated anywhere there's a change.
    - Some changes are pending with VBA stakeholders and need confirmation from SMEs.

### Problems ⚠️
`Notes from KO 11/6/2023`
- **One change** on Step [3](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/99FEF874-74D9-41F1-BD5E-AE7E7E1D768C/canvas).
    - **Previously:** Required to include service history.
    - **Now:** Adding question about discharge papers, so some people can skip service history questions.
    - **Archived Version:** Have an archived version where they set it up to allow veterans to upload discharge papers on that Step 3 page, so curious about our thoughts on that.
- **Question about previous names:**
    - Received feedback that people might have difficulty remembering services rendered under previous name, they're clarifying with SMEs what to do with this (is there a plain language version of this question,what does "services rendered" mean, what information do they really need to receive on this question).

</details>

## Design Review Tasks for CAIA A11ys 

- [This linked Design Review Template](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/vsa/accessibility/templates/design-review.md) is what we _traditionally_ review, when conducting wireframe/lo-fi design reviews for a CAIA A11y Design Review. 
- _Based on the development of the prototype, some of the following may or may not be applicable._ 

### [Link to Final Deliverable](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/teams/CAIA/accessibility/Deliverables/Benefits%20and%20Claims%20%7C%20Lifestage%20Benefits%20Crew%20%7C%20Non-Disability%20Experience/%20530EZ%20Burial%20Form%20Design%20Review%20-%20Steps%201-6)

```[tasklist]
### CAIA a11y Comms
- [x] **KICKOFF** Product team intro and initial reach out 11/6/2023 Kickoff
- [x] **CONNECT** with the product team in Slack re: a11y support starting
- [x] **MIDPOINT PREP:** Assist team with prep for Midpoint via Sketch File Review (Sketch files received - linked in ticket and in task list below.
- [x] **TBD** - Share Midpoint Prep Feedback Deliverable with the product team prior to Midpoint (xx/xx/xxxx) (see deliverable tasks list below)
```
```[tasklist]
### Review Sketch Files for Accessibility
- [x] a11y Sketch Review of Step [1](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/350AC352-727B-4D99-B72B-D466886BE199/canvas)
- [x] a11y Sketch Review of Step [2](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/4871EF16-DFC9-460E-AA1B-7ED24E33B188/canvas)
- [x] a11y Sketch Review of Step [3](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/99FEF874-74D9-41F1-BD5E-AE7E7E1D768C/canvas)
- [x] a11y Sketch Review of Step [4](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/B7F2215E-001C-41F3-B803-9DFD96E7B129/canvas)
- [x] a11y Sketch Review of Step [5](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/E1B6C12E-B2F8-4A50-9CC1-1F44853FA3C6/canvas)
- [x] a11y Sketch Review of Step [6](https://www.sketch.com/s/de782a35-e147-4c32-a2a8-ba53071ec8e7/p/28963E25-18C3-4CDA-A356-BF4FD6A7A5A6/canvas)
```
```[tasklist]
### Deliverable from CAIA a11ys
- [x] **Create** markdown draft of the [Design Review Deliverable Template](https://github.com/department-of-veterans-affairs/va.gov-team/edit/master/teams/CAIA/accessibility/templates/Design%20review%20template.md) to create the final deliverable summary and document findings from the design review.
- [x] **Create** a folder for this team via [Teams > CAIA > Accessibility > Deliverables](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/CAIA/accessibility/Deliverables)
- [x] **Save** the final deliverable in this new folder
- [x] **Link** to the deliverable in this ticket
- [x] **Share** the link in the Slack Channel for the team (found in the about section, at the top of this ticket)
```

**Comment by SarahKay8** [comment]
Final deliverable has been shared to Benefits and claims team. Closing @sara-amanda @coforma-terry @steele-lm
